tensorboard/webapp/BUILD

```diff
@@ -156,15 +156,9 @@ tf_sass_library(
     ],
 )
 
-tf_sass_library(
+alias(
     name = "theme",
-    srcs = [
-        "_palette.scss",
-        "_theme.scss",
-    ],
-    deps = [
-        ":angular_material_theming",
-    ],
+    actual = "//tensorboard/webapp/theme",
```

Comment:
I don't fully understand this alias, isn't it the `actual` field exactly the same?

Reply:
It is used by the internal rule. This change guarantees no breakage when syncing.

Reply:
In that case, can we please add a comment, so that nobody accidentally removes this in OSS?